src/axolotl/utils/data.py

```diff
@@ -905,3 +906,30 @@ def encode_packed_pretraining(
             chunked_data[feature].append(collated_features[feature].squeeze(0))
 
     return chunked_data
+
+
+def load_prepare_dpo_datasets(cfg):
+    def load_split(dataset_cfgs, _cfg):
+        split_datasets: List[Any] = []
+        for i, ds_cfg in enumerate(dataset_cfgs):
+            ds = load_dataset(  # pylint: disable=invalid-name
```

Comment:
I don't think this works if you passing a dataset you have on your local machine right? For reference: https://huggingface.co/docs/datasets/loading

```python 
from datasets import load_dataset
dataset = load_dataset("json", data_files="my_file.json")
```
Would be nice to have the option to pass local datasets. But I could also be me not fully understanding load_dataset